### ISSUE TITLE
PROBE_CALIBRATE

### ISSUE THREAD

**Issue by qkpony** [CobraPi/Klipper-Firmware-FLSUN-QQ-S-Pro#19]
Hello I have used your stock bin file and printer config. I have everything working but when I try to do the probe calibrate it gives me a out of range error after 0.0 for Z. I am sure I am missing something simple. But I am trying to get my Z offset. 
[klippy.log](https://github.com/CobraPi/Klipper-Firmware-FLSUN-QQ-S-Pro-Delta-Printer/files/8253470/klippy.log)

**Comment by qkpony** [comment]
Because I couldn't get it to go down far enough I figured I would try to print and just make the adjustment. But this is what I got for that. Any help would be greatly appreciated.  
!! Move out of range: -0.163 15.935 -3.179 [-3.500]

[klippy_1st_print.log](https://github.com/CobraPi/Klipper-Firmware-FLSUN-QQ-S-Pro-Delta-Printer/files/8262258/klippy_1st_print.log)

**Comment by CobraPi** [comment]
This is a known issue that I've addressed but never got around to updating all the configs. Try adding this parameter to the following section:

[printer]
minimum_z_position: -10

**Comment by qkpony** [comment]
That worked, Thank you very much. Is there anyway to get the stock screen to work and say anything other than complete? lol.

**Comment by conlank** [comment]
There is no way to make the display work. 

If you add this:
```
[static_digital_output display_reset]
pins: !PC6, !PD13
```
to your `printer.cfg` you can turn the display off so it does not display "Completed".

**Comment by qkpony** [comment]
{'message': "horizontal_move_z can't be less than probe's z_offset", 'error': 'WebRequestError'}

I had everything working then had to redo image because my sd card went bad. I got everything working now I have this. my move z is higher than my z offset. my z-offset is 19.503 and I set the move z to 25, 30 etc. Should I start a new thing? thank you so much for the help.
[klippy.log](https://github.com/CobraPi/Klipper-Firmware-FLSUN-QQ-S-Pro-Delta-Printer/files/8288791/klippy.log)
